resources/views/components/ignore-issue.blade.php

```diff
@@ -1,80 +1,37 @@
-@props(['issueUrl'])
+@props(['issueUrl', 'isIgnored'])
 <div x-data="{
-        ignoreModalOpen: false,
-        addIgnoredUrl(issueUrl){
+        toggleIgnoredUrl(issueUrl){
             let ignoredUrls = JSON.parse(localStorage.getItem('ignoreUrl')) ?? [];
-            ignoredUrls.push(issueUrl);
+
+            if (ignoredUrls.includes(issueUrl)) {
+                ignoredUrls = ignoredUrls.filter(ignoredUrl => ignoredUrl !== issueUrl);
+            } else {
+                ignoredUrls.push(issueUrl);
+            }
+
             localStorage.setItem('ignoreUrl', JSON.stringify(ignoredUrls));
 
             getIgnoredUrls();
         }
      }"
 >
     <button
-        @click="ignoreModalOpen = ! ignoreModalOpen"
+        @click="toggleIgnoredUrl('{{$issueUrl}}');"
         class="flex-1 border text-gray-400 hover:bg-gray-100 dark:border-slate-600 dark:hover:bg-slate-600 font-bold px-3 py-2 ml-1 rounded-md outline-none focus:outline-none ease-linear transition-all duration-150"
         type="button"
     >
-        <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24"
-             stroke="currentColor" stroke-width="2">
-            <path stroke-linecap="round" stroke-linejoin="round"
-                  d="M13.875 18.825A10.05 10.05 0 0112 19c-4.478 0-8.268-2.943-9.543-7a9.97 9.97 0 011.563-3.029m5.858.908a3 3 0 114.243 4.243M9.878 9.878l4.242 4.242M9.88 9.88l-3.29-3.29m7.532 7.532l3.29 3.29M3 3l3.59 3.59m0 0A9.953 9.953 0 0112 5c4.478 0 8.268 2.943 9.543 7a10.025 10.025 0 01-4.132 5.411m0 0L21 21"/>
-        </svg>
+        @if($isIgnored)
+            <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor" stroke-width="2">
+                <path stroke-linecap="round" stroke-linejoin="round" d="M15 12a3 3 0 11-6 0 3 3 0 016 0z" />
+                <path stroke-linecap="round" stroke-linejoin="round" d="M2.458 12C3.732 7.943 7.523 5 12 5c4.478 0 8.268 2.943 9.542 7-1.274 4.057-5.064 7-9.542 7-4.477 0-8.268-2.943-9.542-7z" />
+            </svg>
+        @else
+            <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24"
+                 stroke="currentColor" stroke-width="2">
+                <path stroke-linecap="round" stroke-linejoin="round"
+                      d="M13.875 18.825A10.05 10.05 0 0112 19c-4.478 0-8.268-2.943-9.543-7a9.97 9.97 0 011.563-3.029m5.858.908a3 3 0 114.243 4.243M9.878 9.878l4.242 4.242M9.88 9.88l-3.29-3.29m7.532 7.532l3.29 3.29M3 3l3.59 3.59m0 0A9.953 9.953 0 0112 5c4.478 0 8.268 2.943 9.543 7a10.025 10.025 0 01-4.132 5.411m0 0L21 21"/>
+            </svg>
+        @endif
     </button>
 
-    <div x-show="ignoreModalOpen" class="relative z-10" aria-labelledby="modal-title"
-         role="dialog" aria-modal="true">
-        <div class="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity"></div>
```

Comment:
I think that the modal is still helpful for users who might misclick or mistap the ignored icon, even if there is a toggle option to change either show or hide ignored issues.

What do you think about it ? @ash-jc-allen @rubenvanerk 
Thank you.

Reply:
Alpine components has one: https://alpinejs.dev/component/notifications. I could implement that.

Reply:
Nice! I've not got a license for Alpine Components, unfortunately. But, if you have one and it doesn't break any license rules, I'd be happy for you to add the notifications.

If you don't want to though, I'd be happy to add something myself :)